src/logic/function/basic.lean

```diff
@@ -53,6 +53,9 @@ end
 lemma funext_iff {β : α → Sort*} {f₁ f₂ : Π (x : α), β x} : f₁ = f₂ ↔ (∀a, f₁ a = f₂ a) :=
 iff.intro (assume h a, h ▸ rfl) funext
 
+lemma is_symm_op.flip_eq {α β} (op) [is_symm_op α β op] : flip op = op :=
+funext $ λ a, funext $ λ b, (is_symm_op.symm_op a b).symm
```

Comment:
This looks like it's in the wrong namespace to me - I thin it should be `is_symm_op.flip_eq` not `function.is_symm_op.flip_eq`.

Reply:
I moved the lemma out of the `function` namespace.